libtextsecure/account_manager.js

```diff
@@ -545,40 +545,53 @@
         primaryDevicePubKey,
         libloki.crypto.PairingType.REQUEST
       );
-      const authorisation = {
-        primaryDevicePubKey,
-        secondaryDevicePubKey: ourPubKey,
-        requestSignature,
-      };
-      await libloki.api.sendPairingAuthorisation(
-        authorisation,
-        primaryDevicePubKey
+
+      const primaryDevice = libsession.Types.PubKey.from(primaryDevicePubKey);
```

Comment:
`from` can return `undefined` so prefer `new` when you know that it's always going to be a pubkey
```suggestion
      const primaryDevice = new libsession.Types.PubKey(primaryDevicePubKey);
```

Reply:
fixed